exhibitions/src/main/kotlin/edu/artic/exhibitions/AllExhibitionsFragment.kt

```diff
@@ -50,6 +47,16 @@ class AllExhibitionsFragment : BaseViewModelFragment<AllExhibitionsViewModel>()
 
     override fun setupBindings(viewModel: AllExhibitionsViewModel) {
         val adapter = (recyclerView.adapter as AllExhibitionsAdapter)
+
+        /* Ensure search events go through ok. */
+        searchIcon
```

Comment:
can you seperate this out into it's own pull request, i believe AllTours and AllEvents do the same thing so we should update those as well

Reply:
Those other two already use this mechanism - AllExhibitions was the last hold-out. I'll create a PR for that and the layout pieces mentioned below.

Reply:
It's up! Check out #297.